luarpmsort

```diff
@@ -29,17 +29,38 @@ function pkgvercmp(a, b)
 	if vcmp ~= 0 then return vcmp end
 
 	return rpm.vercmp(ar, br)
-end	
+end
 
 vers = {}
 for line in io.stdin:lines() do
 	table.insert(vers, line)
 end
-table.sort(vers, function(a, b) return pkgvercmp(a, b) == 1 end)
+table.sort(vers, function(a, b) return pkgvercmp(a, b) == '"$direction"' end)
 print(table.concat(vers, "\n"))
 '
 
 	rpm --eval "%{lua: ${script}}"
 }
 
-rpmsort
+usage() {
+    cat >&2 <<EOF
+$0 - sort standard input according to rpm version sorting conventions,
```

Comment:
As there's a proper `--help` option now, let's also make it more useful than rpmsort's ever was and add something like `Expects RPM package versions separated by newlines as input and outputs it according to rpm version sorting conventions with old versions at the top.`

Reply:
OK, done. Please check again.